client/src/sections/learning-paths/ContentSection/index.tsx

```diff
@@ -20,55 +20,55 @@ const LearningPathContentSection: FC<LearningPathContentSectionProp> = ({
     <Fragment>
       <div>
         <h3 className="text-lg font-bold text-dark mb-2">Description</h3>
-        <p className="text-sm font-normal text-dark">
-          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ipsum dolores error id, commodi
-          ex et laborum, possimus voluptatum dolore facilis ullam eaque quae a eligendi asperiores
-          placeat accusantium animi blanditiis? Eaque atque ex autem voluptates suscipit! Modi
-          assumenda eum repudiandae nesciunt inventore dignissimos? Doloribus impedit voluptatem
-          quidem assumenda, aspernatur ea.
-        </p>
+        <p className="text-sm font-normal text-dark">{learningPath?.description}</p>
       </div>
       <div className="flex gap-6 mt-4">
```

Comment:
add bottom space

```suggestion
      <div className="flex gap-6 mt-4 mb-4">
```
![image](https://github.com/framgia/sph-lms/assets/115448429/8ec9865f-9785-4616-805a-743902e28a1d)